README.md

```diff
@@ -72,7 +74,7 @@ Once your files are encrypted, they are ready to be submitted to the SDA/BP arch
 
 The configuration file can be downloaded by logging in with a Life Science RI account:
 
-* For BigPicture use [this](https://login.bp.nbis.se/)
+* For BigPicture use https://login.bp.nbis.se/
```

Comment:
I prefer links to be honest, but it's not really a deal breaker :)

Reply:
I am following the instructions Katties presented on the NBIS retreat that one should avoid using [here] or [this] link. But of course these are all personal preferences.